package/src/const/whitelist.ts

```diff
@@ -0,0 +1,113 @@
+import { BridgeType } from './bridges'
+import { ChainType } from './chains'
+
+export enum Token {
```

Comment:
used for UI purposes, would favor the whitelist to be more object based, so that we can define it like:
```
{
  name: 'ampLuna',
  chain: Osmosis,
  denom: 'ibc/abc',
  bridge: ics
}
```

But feedback is welcome, I tried to do it similiar to the whitelist in bridge app

Reply:
We also need some automated way to enable listing new tokens, as this is super complex.

something like list.exe --name ampLuna --token 'terra1ecgazyd0waaj3g7l9cmy5gulhxkps2gmxu9ghducvuypjq68mq2s5lvsct'
and it will send the token through the bridge to all connected chains and will return:

ampLuna = terra...
ampLunaOnJuno = ibc/...
ampLunaOnOsmo = ibc/...

but this is complicated, as right now I manually open a channel with the relayer configuration and send tokens through the channel and look at the transaction on the other side what ibc/... I received.